### PR TITLE
bootstrapped react, testing, linting, created app component

### LINKED ISSUE #1: Bootstrap React component and repository
**What needs to be done? Why does it need to be done?**
Add the basic infrastructure needed to create a React component in version control. The deliverable artifact here is simply a React component that can be dropped into a separate application. However, it's important to be able to test this ourselves, so we need some minimal "app" that we can drop the component into and run _locally_ for testing. 

**Additional context**
We don't need any infrastructure for hosting or deploys. We just need to be able to run/test the component (and sub-components that get created).

### LINKED ISSUE #2: Bootstrap testing and linting
**What needs to be done? Why does it need to be done?**
For this component, we need just basic unit testing and linting. This can follow the standards/conventions from pulse-dashboard. We should be able to run these via `yarn test` and `yarn lint`, respectively. Unlike pulse-dashboard, there should not be a lint whitelist: _every_ file should pass lint in this new repository.

### LINKED ISSUE #3: Create skeleton of the page/components
**What needs to be done? Why does it need to be done?**
First, build the general skeleton of the app as described in the [PRD](https://docs.google.com/document/d/1eOiMzk8_eDjC4W0xn89qlkmj_6SOqi1dBTyEusAGGaI/edit?ts=5fb45edc#) and the [final mocks](https://www.figma.com/file/66HyjH0efZMoAuTlhnojWh/Justice-Counts-Report-Draft?node-id=308%3A3266). This does not need to have full interaction yet and does not need the underlying CSV data file(s). We just want to have the basic UI in place (everything below the header) with the requisite sub-components that we intend to fill in, and want to ensure that the basic UI looks reasonable in the browsers/environments documented in the PRD.

Ultimately, there are 3 separate components to be built here:
- "Key insights" at the top of the page (below the header we don't need to build)
- The "Flowchart" that comes next
- Line graphs that show certain metrics over time at the bottom of the page (above the footer that we don't need to build)
  - There are multiple graphs that can be selected from to show in  (the exact set of graphs is still to be finalized in the PRD)

**Additional context**
Later tasks will fill in the interactive functionality and data usage.

### PR DESCRIPTION
**Pull request by Wmaileq**
## Description of the change

- bootstrapped app skeleton, linting and testing utils
- added fonts and created UI for the main page
- added initial test

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [X] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Configuration change (adjusts configuration to achieve some end related to functionality, development, performance, or security)

## Related issues

> Closes #1 , Closes #2 , Closes #3 

### Development

These boxes should be checked by the submitter prior to merging:

- [X] Manual testing against realistic data has been performed locally

### Code review

These boxes should be checked by reviewers prior to merging:

- [x] This pull request has a descriptive title and information useful to a reviewer
- [x] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [x] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

**Review by jessex — CHANGES_REQUESTED** [review]
Off to a great start, @Wmaileq! Just two minor formatting notes for you.

**Comment by Wmaileq** [comment]
@jessex Percent in the card made with 500 weight font, so need corresponding font file (supervision success had only 400 and 700)

**Review by jessex — COMMENTED** [review]
Looking good so far, @Wmaileq! Are you planning to do more in this initial PR (e.g. add skeletons for the graphs, or add the rest of the arrows to the flowchart) or is this done for now?

Regarding the fonts, which font file do you need exactly? Now that you have comment abilities on the PRD, would you mind asking Humphrey there directly?

**Comment by Wmaileq** [comment]
@jessex I am thinking about adding all the UI in this PR.

**Review by jessex — CHANGES_REQUESTED** [review]
Thanks, @Wmaileq! A few comments for your latest batch here.

Also, I see the following errors in the console logs that we'll want to take care of before merging:

Screen Shot 2020-12-02 at 12 53 58

File: src/FlowDiagram/FlowDiagram.js [hunk 1]
```diff
@@ -15,53 +15,68 @@ const FlowDiagram = ({ lastDate, prevDate }) => (
...
           className="FlowDiagram__card"
-          title="Probation population"
+          title="Probation Population"
           number={68285}
           percent={-8}
           isPopulation
           warning="Probation population was last reported on June 30, 2020 (% change compared to September 2019)."
-        />
+        >
+          <Arrow />
```
[the remark this answers is not in the document]
Comment (Wmaileq) [reply src/FlowDiagram/FlowDiagram.js:37]: Now arrows color goes from cards. `Card__arrow` class. It makes all arrows gray when the card itself disabled. 
But seems like it is not a working solution, I will introduce a new prop in the arrow itself.

**Review by jessex — APPROVED** [review]
Looks great, @Wmaileq! Well done. Are you planning to continue more work on this branch or am I clear to merge this one?

**Comment by Wmaileq** [comment]
@jessex feel free to merge this.

**Pull request merged**